golang/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/google/uuid"
+	"github.com/momentohq/client-sdk-go/momento"
+)
+
+func main() {
+	var AuthToken = os.Getenv("MOMENTO_AUTH_TOKEN")
+	const (
+		CacheName             = "momentocache"
```

Comment:
maybe make this env var you pass in similar to auth token. I think this is consistent w/ other examples so customer can easily change cache name

Reply:
Oh it looks like other SDK examples are setting cache name as a const variable?
https://github.com/momentohq/client-sdk-examples/blob/main/javascript/index.ts#L3
https://github.com/momentohq/client-sdk-examples/blob/main/python/example.py#L7

Reply:
ok then lets keep consistent for now